config/C4BB Client Search Tests.postman_collection.json

```diff
@@ -393,7 +393,7 @@
 				"method": "GET",
 				"header": [],
 				"url": {
-					"raw": "{{url_prefix}}/ExplanationOfBenefit?_lastUpdated=2024-10-21",
+					"raw": "{{url_prefix}}/ExplanationOfBenefit?_lastUpdated>2024-10-21",
```

Comment:
This is not the correct form for this search. FHIR parameters always use = to distinguish between the parameter name and value. To do something other than equality comparison, you use [prefixes](https://hl7.org/fhir/R4/search.html#prefix). Here it would be `_lastUpdated=gt2024-10-21`.